wgpu/src/lib.rs

```diff
@@ -1826,23 +1826,34 @@ impl Instance {
     ///
     /// # Safety
     ///
-    /// - Raw Window Handle must be a valid object to create a surface upon and
-    ///   must remain valid for the lifetime of the returned surface.
-    /// - If not called on the main thread, metal backend will panic.
+    /// - `raw_window_handle` must be a valid object to create a surface upon.
+    /// - `raw_window_handle` must remain valid until after the returned [`Surface`] is
+    ///   dropped.
+    ///
+    /// # Errors
+    ///
+    /// - On WebGL 2: Will return an error if the browser does not support WebGL 2,
```

Comment:
Minor nit: `WebGL2` is how we generally style it.

Reply:
I just mean internally consistent with ourselves like https://docs.rs/wgpu-types/latest/wgpu_types/struct.DownlevelFlags.html#associatedconstant.COMPUTE_SHADERS. Ultimately doesn't matter.

Reply:
… done.